L1Trigger/TrackTrigger/python/Setup_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
This is an important cfg file. It deserves a comment at the top explaining what it represents.

Reply:
done